argparse/argparse.py

```diff
@@ -3,7 +3,10 @@
 """
 
 import sys
-from _collections import namedtuple
+try:
+    from _collections import namedtuple
+except ImportError:
+    from collections import namedtuple
```

Comment:
I thought it would be valuable to make a module work with both uPy and CPython when possible but it's not a requirement for me. Making it work with CPython was a side-effect of trying to figure if some unexpected behaviour I was seeing was caused by uPy.